### DISCUSSION TITLE
Tailwind classes inexplicably being applied to docs in Docusaurus

### DISCUSSION THREAD

**Discussion by franko1260** [firecmsco/firecms#324] [Q&A]
Hey team,

With the website that you built using Docusaurus, I quite like your approach of using Tailwind for the `index.js` page of the site. I have found however when I tried this approach for myself I found that Tailwind classes were inexplicably being applied to Docusaurus components that go well beyond the scope of `index.js`, particularly the docs:

![image](https://user-images.githubusercontent.com/16366632/171833829-8ece19a3-a0ea-45a1-89ae-e52942de1c45.png)

After troubleshooting this for over 2 weeks (which at this point I feel as though I know the source code of your website like the back of my hand), I'm at a complete loss as to why this is happening. If you came across this problem yourselves, how the heck did you fix it?

**Comment by fgatti675** [comment]
Sorry @franko1260 
We didn't run into this issue! It is definitely weird
Feel free to use our website as a template if it helps
